### ISSUE TITLE
Container creation fails for versions 0.32.x and above

### ISSUE THREAD

**Issue by pjanouse** [crossplane-contrib/provider-aws#1786]
I can't re-open the original report thus creating the new one.

The issue reported at #1531 isn't fixed yet. Verified again right now and version v.0.41.0 released couple of days ago is still affected. The last known working version is still v0.31.0 (`crossplane/provider-aws:v0.31.0`).

**Comment by MisterMX** [comment]
@pjanouse can you describe the issue you are having and provide a way to replicate it? From what I see in the linked issue it seems like a system misconfiguration to me that is not linked to the provider itself. We don't experience any issue regarding container creation.

**Comment by pjanouse** [comment]
Hello,
I've lost an access to my original system(s) where I was able to reproduce the issue (originally reported by other guy as #1531) - I was under the pressure of the same issue for couple of Crossplane providers in my own environment, so tried to solve my own issue by searching over the web and it brought me to #1531. At the time of opening this report (#1786) the issue was still here, I was able to reproduce it quite easily with minikube, ArgoCD and (AWS but not only) Crossplane provider(s) (the latest version at that time), so I was forced to stay with the old (working) version in my environment.

Anyway, **good news for today** - when started from scratch this week and using the most fresh version (means published versions/releases) of *all* components for the whole stack (from kernel, OS (Fedora), through k8s cluster (minikube), applications (like ArgoCD) and Crossplane providers) and now it seems working without any issue. 

I'm closing this issue because not reproducible now with the latest published components.
